### PR TITLE
Make istio gateway configurable in tensorboard-controller

### PR DESCRIPTION
**Pull request by haijohn**
this PR make istio gateway in tensorboard-controller configurable, like that in notebook-controller

### CODE REVIEW

File: components/tensorboard-controller/config/manager/kustomization.yaml [hunk 1]
```diff
@@ -1,2 +1,6 @@
 resources:
 - manager.yaml
+configMapGenerator:
+- name: config
+  literals:
+  - ISTIO_GATEWAY=kubeflow/kubeflow-gateway
```
Comment (DavidSpek) [inline components/tensorboard-controller/config/manager/kustomization.yaml:6]: Nit: Please add a new line here

**Comment by DavidSpek** [comment]
@haijohn Thanks for your contribution. When looking at the notebook-controller as a reference the configmap renerator would also need to be added to https://github.com/kubeflow/kubeflow/blob/master/components/tensorboard-controller/config/overlays/kubeflow/kustomization.yaml.

**Comment by haijohn** [comment]
@DavidSpek  After put configMapGenerator in https://github.com/kubeflow/kubeflow/blob/master/components/tensorboard-controller/config/overlays/kubeflow/kustomization.yaml. I failed to run  `kustomize build config/overlay/kubeflow` , because there is a confilict with the configmap generator in base dir. To be consistnent, I moved ISTIO_GATEWAY env to https://github.com/kubeflow/kubeflow/blob/master/components/tensorboard-controller/config/base/kustomization.yaml

**Pull request closed without merging**